src/SerialLoops/Editors/MapEditor.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Eto.Forms;
+using SerialLoops.Lib.Items;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SerialLoops.Editors
+{
+    internal class MapEditor : Editor
+    {
+
+        private MapItem _item;
```

Comment:
Probably can name this more descriptively

```suggestion
        private MapItem _map;
```